Makefile

```diff
@@ -1,9 +1,10 @@
 # Override the arch with `make ARCH=i386`
-ARCH   ?= $(shell flatpak --default-arch)
-REPO   ?= repo
+ARCH    ?= $(shell flatpak --default-arch)
+REPO    ?= repo
+FB_ARGS ?= --user --sandbox --delete-build-dirs --install-deps-from=flathub
```

Comment:
Some of these args are duplicated in the call to `flatpak-builder`.

Reply:
Good catch. I fixed it locally but I'll wait on more feedback before I force-push.